pkg/model/project.go

```diff
@@ -351,51 +351,31 @@ func (p *ProjectSSOConfig_GitHub) GenerateAuthCodeURL(project, callbackURL, stat
 	return authURL, nil
 }
 
-// FilterRBACRoles filter rbac roles for built-in or not.
-func (p *Project) FilterRBACRoles(isBuiltin bool) []*ProjectRBACRole {
-	v := make([]*ProjectRBACRole, 0, len(p.RbacRoles))
-	for _, role := range p.RbacRoles {
-		if role.IsBuiltin != isBuiltin {
-			continue
+// HasRBACRole checks whether the RBAC role is exists.
+func (p *Project) HasRBACRole(name string) bool {
+	for _, v := range p.RbacRoles {
+		if v.Name == name {
+			return true
 		}
-		v = append(v, role)
 	}
-	return v
+	return false
 }
 
-// ValidateRBACRoles validate rbac roles.
-func ValidateRBACRoles(roles []*ProjectRBACRole) error {
-	v := make(map[string]struct{}, len(roles))
-	for _, role := range roles {
-		if role.IsBuiltinName() {
-			return fmt.Errorf("cannot use built-in role name")
+// IsRBACRoleAssigned checks whether the user group is exists.
+func (p *Project) IsRBACRoleAssigned(sso string) bool {
```

Comment:
How about `HasUserGroup`? To be unified with `HasRBACRole`.

Reply:
Sorry, you've lost me.
I think that `HasUserGroup` and `HasRBACRole` are different as I mentioned in [this](https://github.com/pipe-cd/pipecd/pull/3581#discussion_r859295585) before.

Reply:
I meant renaming `IsRBACRoleAssigned` to `HasUserGroup`.

Because when I see the comments of `HasRBACRole` and `IsRBACRoleAssigned` they are saying something similar so their name should be unified as well. (I didn't mean that they should be merged and I see they are different things, just want to change the name of `IsRBACRoleAssigned` one.)

- HasRBACRole **checks whether the RBAC role exists.**
- IsRBACRoleAssigned **checks whether the user group exists.**

Reply:
Thank you for your explanation and got your point.
I addressed by https://github.com/pipe-cd/pipecd/pull/3581/commits/b8e212b062040f49196b6b34b4f88e20ae376900